<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGuestMainWindow</class>
 <widget class="QMainWindow" name="NewGuestMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Guests</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    selection-background-color: darkgray;
}

QDateTimeEdit {
  border: 2px solid gray;
  border-radius: 5px;
}

QTextEdit {
  border: 2px solid gray;
  border-radius: 5px;
}
QPushButton {
   border: 2px solid;
   border-radius: 6px;
   padding: 6px;
}

QComboBox {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(/usr/share/icons/crystalsvg/16x16/actions/1downarrow.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}

QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #C2C7CB;
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
}

QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="new_guest_tab">
       <attribute name="title">
        <string>New Guest</string>
       </attribute>
       <widget class="QLineEdit" name="surname_edit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Surname</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>20</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Other names</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>60</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nationality</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Occupation</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nationality_edit">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>60</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="other_names_edit">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>20</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="occupation_edit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>125</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="address_tedit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>191</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Address in full</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>245</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Vehicle info</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="phone_number_edit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>160</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="vehicle_edit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>235</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>207</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Means of ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_card_edit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>197</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>100</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Next of kin name</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="nok_name_edit">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>100</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="nok_address_tedit">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>190</y>
          <width>201</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>210</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Next of kin address</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="nok_phone_edit">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>140</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>140</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Next of kin phone number</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="next_button">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>290</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="passport_tab">
       <attribute name="title">
        <string>Passport and other metadata</string>
       </attribute>
       <widget class="QLabel" name="upload_passport_label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>371</width>
          <height>291</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>340</y>
          <width>371</width>
          <height>66</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="use_camera_button">
           <property name="text">
            <string>&amp;Use Camera</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="upload_passport_button">
           <property name="text">
            <string>Upload from computer</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="save_button">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="find_guest_tab">
       <attribute name="title">
        <string>Find Existing Guest</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Search term</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="search_string_edit"/>
        </item>
        <item row="2" column="0">
         <widget class="QComboBox" name="criteria_combo"/>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QDateEdit" name="dateEdit">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QPushButton" name="find_button">
          <property name="text">
           <string>Find</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>543</width>
             <height>312</height>
            </rect>
           </property>
           <widget class="QLineEdit" name="other_names_edit_2">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>46</y>
              <width>241</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_23">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>11</y>
              <width>42</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Surname</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="surname_edit_2">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>10</y>
              <width>241</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_21">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>62</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Other names</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>400</x>
              <y>10</y>
              <width>121</width>
              <height>131</height>
             </rect>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton">
            <property name="geometry">
             <rect>
              <x>246</x>
              <y>120</y>
              <width>75</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_3">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>120</y>
              <width>75</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="other_names_edit_3">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>81</y>
              <width>241</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_22">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>85</y>
              <width>62</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="order_and_bills_tab">
       <attribute name="title">
        <string>Orders and Bills</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="sizeConstraint">
                <enum>QLayout::SetNoConstraint</enum>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_31">
                   <property name="text">
                    <string>Room number</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_2"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_32">
                   <property name="text">
                    <string>Item purchased on</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="wordWrap">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateTimeEdit" name="dateTimeEdit_4">
                   <property name="showGroupSeparator" stdset="0">
                    <bool>true</bool>
                   </property>
                   <property name="currentSection">
                    <enum>QDateTimeEdit::MonthSection</enum>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_30">
                   <property name="text">
                    <string>Today's date</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="wordWrap">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateTimeEdit" name="dateTimeEdit_3">
                   <property name="showGroupSeparator" stdset="0">
                    <bool>true</bool>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_28">
                   <property name="text">
                    <string>Room number</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QPushButton" name="add_order">
                 <property name="text">
                  <string>Add Item</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="edit_order">
                 <property name="text">
                  <string>Edit Item</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="delete_order">
                 <property name="text">
                  <string>Delete Item</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QTableView" name="tableView"/>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Print</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
